hack/testing-olm-upgrade/secrets/kibana-trusted-ca-bundle.yaml

```diff
@@ -0,0 +1,3681 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
we shouldn't need to create this for the test... do we? this should be handled by k8s and this operator

Reply:
As far as I remember these certs and configmaps are generated by the CLO, not the EO.

Reply:
these configmaps are not, EO will create them if there is a proxy config

Reply:
Fixed